tests/product/test_configuration.py

```diff
@@ -221,23 +218,6 @@ def test_lost_coordinator_connection(self):
         self.assertRegexpMatches(str.join('\n', output.splitlines()[6:]),
                                  expected)
 
-    def test_deploy_lost_worker_connection(self):
```

Comment:
there's already a test for a lost coordinator connection and we don't need both.